torch/_inductor/codegen/cuda/cuda_kernel.py

```diff
@@ -157,25 +166,55 @@ def call_kernel(
         as well as all required inputs and outputs.
         """
         wrapper = V.graph.wrapper_code
-        _, call_args, _, arg_types = self.args.python_argdefs()
+
+        if V.graph.cpp_wrapper:
```

Comment:
Stylistic nit: looks like two `if` blocks can be merged?

Reply:
yes good call!